client/src/Auth/Register/register.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+// https://www.flaticon.com/free-icon/circular-avatar_108331
+import avatar from './avatar.png';
+import './register.css';
+
+// Checks whether state values meets valid criteria
+function validateInputs ({ username, password, confirmPassword }) {
+    
+    return {
+        username: username.length < 4,
+        password: password.length < 5,
+        confirmPassword: confirmPassword !== password
+    }
+}
+
+// If an input is not focused has value but not complete returns false
+function checkDirty (focus, {username, password, confirmPassword}, errors) {
+
+    return {
+        username: focus !== 'username' && errors.username && username.length > 1,
+        password: focus !== 'password' && errors.password && password.length > 1,
+        confirmPassword: focus !== 'confirmPassword' && errors.confirmPassword && password.length > 1
+    }
+}
+
+const Register = (props) => {
+
+    const { focus } = props.inputs;
+    const errors = validateInputs(props.inputs);
+    const isDirty = checkDirty(focus, props.inputs, errors);
+    const isEnabled = Object.keys(errors).some(e => errors[e]);
+
+    return (
+        <div className="container-fluid d-flex flex-column my-auto">
+                <div className="d-flex flex-column col-lg-5 col-md-3 col-sm-8 mx-auto justify-content-center register-wrapper">
+                    <div className="align-center pt-2 mb-3">
+                        <h2 className="register-text">Register</h2>
+                    </div>
+                    <img src={avatar} className="align-self-center my-3" width="120px" />
```

Comment:
You have an ESLing warning here: It needs an alt attribute, probably empty as it's just to pretty up the page.